src/operator/elementwise_unary_op-inl.h

```diff
@@ -128,6 +129,26 @@ MXNET_REGISTER_SIMPLE_OP(log, XPU)
 .set_function(XPU::kDevMask, UnaryForward_<XPU, mshadow_op::log>, kInplaceInOut)
 .set_gradient(XPU::kDevMask, UnaryBackwardUseIn_<XPU, mshadow_op::log_grad>, kInplaceOutIn)
 .describe("Take log of the src");
+// log1p
+MXNET_REGISTER_SIMPLE_OP(log1p, XPU)
+.set_function(XPU::kDevMask, UnaryForward_<XPU, mshadow_op::log1p>, kInplaceInOut)
+.set_gradient(XPU::kDevMask, UnaryBackwardUseIn_<XPU, mshadow_op::log1p_grad>, kInplaceOutIn)
+.describe("Take log1p of the src");
+// expm1
+MXNET_REGISTER_SIMPLE_OP(expm1, XPU)
+.set_function(XPU::kDevMask, UnaryForward_<XPU, mshadow_op::expm1>, kInplaceInOut)
+.set_gradient(XPU::kDevMask, UnaryBackwardUseIn_<XPU, mshadow_op::exp>, kInplaceOutIn)
+.describe("Take expm1 of the src");
```

Comment:
since these are less common functions, please explain what they do. Best if you can put formula here